ton_client/src/proofs/mod.rs

```diff
@@ -305,7 +407,8 @@ impl BlockProof {
             )))?;
         let _cur_validator_set = config.config(34)?
             .ok_or_else(|| Error::invalid_data(format!(
-                "proof for key block {} contains a Merkle proof without current validators config param (34)",
+                "proof for key block {} contains a Merkle proof without current validators config \
```

Comment:
No hash check for key blocks. Need to add it

Reply:
It was checked before in `virtualize_block()`.